openassessment/xblock/studio_mixin.py

```diff
@@ -147,9 +148,9 @@ def editor_context(self):
         white_listed_file_types_string = ','.join(white_listed_file_types) if white_listed_file_types else ''
 
         # If rubric reuse is enabled, include information about the other ORAs in this course
-        other_openassessments_in_course = []
+        rubric_reuse_data = {}
         if self.is_rubric_reuse_enabled:
-            other_openassessments_in_course = self.get_other_ora_blocks_for_rubric_editor_context()
+            rubric_reuse_data = self.get_other_ora_blocks_for_rubric_editor_context()
```

Comment:
`self.get_other_ora_blocks_for_rubric_editor_context()` is still returning `list` and not `dictionary` right?

Reply:
It's a dict. I like the idea of leaving it because some day I want the way we display this to change and then we won't have to touch the backend